charts/longhorn-crd/103.1.1+up1.4.4/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+annotations:
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/hidden: "true"
+  catalog.cattle.io/namespace: longhorn-system
+  catalog.cattle.io/release-name: longhorn-crd
```

Comment:
`longhorn-crd` is a dependent chart of the main chart `longhorn`. I am wondering if we also need to add `catalog.cattle.io/permits-os: linux,windows` annotation @rohitsakala @lucasmlp

Reply:
@PhanLe1010 no need since longhorn crd is tied to longhorn chart and longhorn crd chart is not visible to customers.